tools/realsense-viewer/realsense-viewer.cpp

```diff
@@ -332,9 +332,12 @@ int main(int argc, const char** argv) try
 
     std::shared_ptr<device_models_list> device_models = std::make_shared<device_models_list>();
 
-    nlohmann::json settings;
+    nlohmann::json settings = nlohmann::json::object();
     if( only_sw_arg.getValue() )
+    {
+        settings["dds"]["enabled"] = true;  // override global dds:false or dds/enabled:false, if any
```

Comment:
I think this should be true only if BUILD_WITH_DDS is set

Reply:
Doesn't hurt, but OK.
Do you think it makes sense to enable DDS automatically if we see "sw-only" device-mask (i.e., override enabled=false if device-mask is sw-only)?

Reply:
No, could be playback or future type devices. If settings set to disable then leave disabled